Sources/TSCBasic/Process.swift

```diff
@@ -229,19 +232,48 @@ public final class Process: ObjectIdentifierProtocol {
     ///   - arguments: The arguments for the subprocess.
     ///   - environment: The environment to pass to subprocess. By default the current process environment
     ///     will be inherited.
+    ///   - workingDirectory: The path to the directory under which to run the process.
     ///   - outputRedirection: How process redirects its output. Default value is .collect.
     ///   - verbose: If true, launch() will print the arguments of the subprocess before launching it.
     ///   - startNewProcessGroup: If true, a new progress group is created for the child making it
     ///     continue running even if the parent is killed or interrupted. Default value is true.
+    @available(macOS 10.15, *)
     public init(
         arguments: [String],
         environment: [String: String] = ProcessEnv.vars,
+        workingDirectory: AbsolutePath? = nil,
```

Comment:
Hm, isn't this initializer ambiguous with the other one? Interesting if it's working fine on macOS.

Reply:
updated to make it less ambiguous and to account for linux not having the working directory required method